### DISCUSSION TITLE
Which options are saved?

### DISCUSSION THREAD

**Discussion by AardeGeoWorld** [Stellarium/stellarium#3105] [Q&A] [answered]
There is two options in Stellarium to save. But what exactly gets saved and what not? For instance, if I change constellation line widths, select specific DSO types and turn off satellites, is that all saved?

**Comment by gzotti** [comment, marked as answer]
Some settings like color are stored immediately. Most other settings require explicit saving. Your first 2 settings should be covered by pressing "Save settings" in the Configuration dialog. The Satellites plugin has its own "Save settings" button.

**Reply by AardeGeoWorld** [reply]
Ah, thank you Georg. I tested it and this seems to work well. Great!
